backend/src/modules/doubles-tournament/core/DoublesTournamentModuleCore.ts

```diff
@@ -50,9 +59,25 @@ export function DoublesTournamentModuleCore(
         eventType: PlayersWereMatchedIntoTeams,
         handler: new CreateTournamentWhenPlayersWereMatchedIntoTeams(commandPublisher),
       },
+      {
+        eventType: MatchWasQueued,
+        handler: new CallMatchWhenMatchWasQueued(commandPublisher, matchesQueue, tablesQueue),
+      },
       {
         eventType: MatchWasCalled,
-        handler: new StartMatchAfterItsCalling(commandPublisher),
+        handler: new StartMatchAfterItsCalling(commandPublisher, matchesQueue),
+      },
+      {
+        eventType: TournamentTableWasReleased,
```

Comment:
Nie ogarniam trochę tych dwóch handlerów.
1. Zwalnia stół w kolejce
2. Wywołuje mecz na stole z kolejki.

Co jeśli szybciej wykona się 2. To potem 1. zwoli stół, który został zajęty? 
Jak to działa :) ?

Reply:
Nie wiem, jak to inaczej ugryźć, skoro:
a) nie da rady asynchronicznie,
b) nie mogę wrzucić tego do jednego handlera.
Teoretycznie startuje najpierw zwalnianie stołu, a dopiero potem wywoływanie meczu. Stół będzie zajęty dopiero po wywołaniu meczu i otrzymaniu eventu o jego zajęciu, co powinno nastąpić później (ale nie daję gwarancji :P).